main.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>Pokedex</title>
```

Comment:
You should add '<meta name="viewport" content="width=device-width, initial-scale=1.0">' to set your viewport width on load. if you go into your console and try to change the responsive width of the screen in this mode, you will notice that your different media queries don't take effect.